### PR TITLE
Cloud credential data source

### PR DESCRIPTION
**Pull request by rawmind0**
Fix PR #11

### CODE REVIEW

File: website/docs/d/cloudCredential.html.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+layout: "rancher2"
+page_title: "Rancher2: rancher2_cloud_credential"
+sidebar_current: "docs-rancher2-datasource-cloud_credential"
+description: |-
+  Get information on a Rancher v2 Cloud Credentials.
+---
+
+# rancher2\_cloud\_credential
+
+Use this data source to retrieve information about a Rancher v2 setting.
```
Comment (cloudnautique) [inline website/docs/d/cloudCredential.html.markdown]: Not really a setting.

Rancher v2 cloud credential tied to a user. maybe?

File: website/docs/d/cloudCredential.html.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Example Usage
+
+```
+data "rancher2_cloud_credential" "test" {
+    name = "test"
```
Comment (cloudnautique) [inline website/docs/d/cloudCredential.html.markdown:17]: Is the name property unique per user/access key scope?

File: website/docs/d/cloudCredential.html.markdown (hunk 1 above)
Comment (rawmind0) [reply website/docs/d/cloudCredential.html.markdown]: yep, it's a typo. Fixed.

File: website/docs/d/cloudCredential.html.markdown (hunk 2 above)
Comment (rawmind0) [reply website/docs/d/cloudCredential.html.markdown:17]: Yep, cloud-credential name is unique.

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**